jest-puppeteer.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     launch: {
         dumpio: true,
-        headless: "new",
+        headless: true,
```

Comment:
why modifying headless: "new"

I remember of a warning during test executions blaming to use headless new in place of deprecated headless true